docs/witness_sign.md

```diff
@@ -22,6 +22,7 @@ witness sign [file] [flags]
   -k, --signer-file-key-path string                   Path to the file containing the private key
       --signer-fulcio-oidc-client-id string           OIDC client ID to use for authentication
       --signer-fulcio-oidc-issuer string              OIDC issuer to use for authentication
+      --signer-fulcio-oidc-redirect-url string        OIDC redirect URL (Optional). The default oidc-redirect-url is 'http://localhost:0/auth/callback'.
```

Comment:
Same comment as above.

Reply:
This is fine right? I think i'm correct in saying that this change to the docs is a consequence of `go-witness` getting bumped.

Reply:
Looks that way since the make docsgen check passed. 😅